index.html

```diff
@@ -1,177 +1,196 @@
+            <li class="category flex cross-center ellipse line-height highlight uppe">Back End Dev</li>
+            <li class="category flex cross-center ellipse line-height highlight uppe">2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+          <ul class="tags flex">
+            <li class="line-height round-borders">html</li>
+            <li class="line-height round-borders">css</li>
+            <li class="line-height round-borders">javaScript</li>
+          </ul>
+          <button class="bg round-borders cap" type="button">see project</button>
+        </div>
+      </article>
+    </div>
+  </section>
+
+  <section class="aboutMe margin-seprator bg">
+    <div class="wrapper hero flex space-between flex-column">
+      <div class="left-side hero flex space-between flex-column">
+        <h2 class="heading">About
+          Myself</h2>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my
+          work
+          and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="#">Let's contact</a>
+        <ul class="social">
+          <li><a href="#"><img src="images/social/twitter.svg" alt="twitter"></a></li>
+          <li><a href="#"><img src="images/social/linkedin.svg" alt="linkedin"></a></li>
+          <li><a href="#"><img src="images/social/medium.svg" alt="medium"></a></li>
+          <li><a href="#"><img src="images/social/github.svg" alt="github"></a></li>
+          <li><a href="#"><img src="images/social/vector.svg" alt="vector"></a></li>
+        </ul>
+        <button class="bg round-borders cap resume" type="button">Get my resume</button>
+      </div>
+
+      <details class="ability" open>
+        <summary>
+          <h3> Languages</h3>
+          <img src="images/arrow.svg" alt="arrow">
+        </summary>
+
+        <ul class="languages">
+          <li class="lang flex cross-center round-borders"><img src="images/lingos/javascript.svg"
+              alt="javascript"><span>JavaScript</span></li>
+          <li class="lang flex cross-center round-borders"><img src="images/lingos/html.svg"
+              alt="html"><span>html</span></li>
+          <li class="lang flex cross-center round-borders"><img src="images/lingos/css.svg" alt="css"> <span>css</span>
+          </li>
+        </ul>
+      </details>
+
+      <details class="abitity">
+        <summary>
+          <h3> Frameworks</h3>
+          <img src="images/arrow.svg" alt="arrow">
+        </summary>
+      </details>
+
+      <details class="abitity">
+        <summary>
+          <h3>Skills</h3>
+          <img src="images/arrow.svg" alt="arrow">
+        </summary>
+      </details>
+    </div>
+  </section>
+  <footer>
+    <div class="footer">
+      <h2 class="heading bold cap">Contact me</h2>

```

Comment:
- [ ] (Optional) It would be nice to use the semantic tag `section` instead of `footer` for this element. Ideally, the `footer` of any project should contain _**copyright, logo, author, rights reserved, and links to your various social media**_. 😄